src/components/victory-chart.jsx

```diff
@@ -425,10 +429,21 @@ export default class VictoryChart extends React.Component {
     return _formatData(childData);
   }
 
+  determineCategoryIndex(x, categories) {
```

Comment:
I think there's more stuff like that. Do you mind if I do a full audit as a separate ticket?